### DISCUSSION TITLE
An attempt to brainstorm the goal

### DISCUSSION THREAD

**Discussion by rkzm** [rarely-org/rarely-mobile#2] [General]
👋🏿 Thanks for raising this issue @steffen. I hope my understanding is correct that you raised the goal as a [discussion](https://github.com/rarely-org/rarely/discussions/1) so that we can do just that - discuss 😍  

I have a few clarifying questions just to make sure the scope or target audience is clear.

> AS A affected or caregiving person 

As `a part of a community`, `a family`, `a neighbor`, `a friend` or `a company`, `a donor`, `a consultant` (just to mention but a few) - do any of these personas apply? What is the scope of affected or care-giving person.

> I WANT TO discover new aids

It the application meant for discovering new aids (only) or help? Should we focus more on sharing experiences and let discovering aids be a by product?

> SO THAT I can improve the quality of my life

I think this is a broad enough goal -  we just need to find a way we can measure it.

Awesome start!!

**Comment by steffen** [comment]
👋 @rkzm First, thank you so much for your amazing points and perspective! 😍 I have not forgotten about this! I have been thinking about it for the past weeks! 😊

> > AS A affected or caregiving person 

> As a part of a community, a family, a neighbor, a friend or a company, a donor, a consultant (just to mention but a few) - do any of these personas apply? What is the scope of affected or care-giving person.

💯 The more I think about it, the more I see that Rarely may have value to people outside of the "affected or caregiving person" group, e.g. teachers/doctors/specialists and family/friends/colleagues to add a few to your list. In general it will be open to everyone.
I wonder though how we specify that in the high-level user stories. 🤔

> > I WANT TO discover new aids

> It the application meant for discovering new aids (only) or help? Should we focus more on sharing experiences and let discovering aids be a by product?

Very good point. I envision a discovery feed that mixes new aids and experiences. Based on type and amount of content and based on what users would like to see we probably need to make it smarter going forward. And yes, maybe it's more experiences and discovering aids is a by-product. ❤️ 

> > SO THAT I can improve the quality of my life

> I think this is a broad enough goal - we just need to find a way we can measure it.

Very very good point! 🙂 If something, e.g. a new aid that they discovered though Rarely, improved someone's life then we don't immediately know it. Either we know about it implicitly by positive experiences that are written and shared within Rarely that hint at discovering it via Rarely or we think about an explicit way to let users share that something in their life has improved thanks to Rarely.
